docs/source/dev_guide/algorithms/dbscan/computation-batch.rst

```diff
@@ -53,6 +53,9 @@ The DBSCAN clustering algorithm has the following parameters:
      - If flag is set to false, all neighborhoods will be computed and stored prior to clustering.
        It will require up to :math:`O(|\text{sum of sizes of all observations' neighborhoods}|)` of additional memory, 
        which in worst case can be :math:`O(|\text{number of observations}|^2)`. However, in general, performance may be better.
+
+       .. note:: ``memorySavingMode`` could be ``false`` only on CPU.
+
```

Comment:
For current implementation, weights are ignored on GPU. Support for weights will be added later

Reply:
fixed